egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -317,6 +317,24 @@
                   "headers": []
                 }
               ]
+            },
```

Comment:
@bhanuprakash-egov Could you please add this chart as a new row, so that we can restrict the number of bar charts per row to "1"